oak/common/fuzzer.bzl

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2019 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def oak_fuzzer(name, srcs = [], deps = [], **kwargs):
```

Comment:
Need a docstring (according to Travis).

Worth including a `corpus = []` explicitly named parameter?

Reply:
Yeah I will add corpus if and when we actually use it :)